api/state_actuals_summary.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+import pydantic
+
+
+class _CountyCases(pydantic.BaseModel):
+    fips: str = pydantic.Field(..., description="FIPS state code + FIPS county code")
+    cases: int = pydantic.Field(..., description="Cumulative case count.")
+    deaths: int = pydantic.Field(..., description="Cumulative deaths count.")
+    date: str = pydantic.Field(..., description="Date of latest data")
```

Comment:
FWIW- This should be an ISO 8601 date string, so we should put that in the description, and then make sure that it is.